src/components/cultural-courses/social-media-communication/LessonComponent.jsx

```
@@ -0,0 +1,442 @@
+
+    setScore(currentScore);
+    setShowResults(true);
+  };
+
+  const resetExercise = () => {
+    setSelectedAnswers({});
+    setShowResults(false);
+    setScore(0);
+  };
+
+  const isAnswerCorrect = (exerciseIndex, questionIndex) => {
```

ESLint

[error] 'isAnswerCorrect' is assigned a value but never used.

(no-unused-vars)